nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, "Subclasses must implement the 'correct_name' method."
+  end
+end
```

Comment:
- You have done great so far :+1: But there are a few changes you need to make before moving to the next milestone. :smile: :wave: 

- [ ] According to the project requirement, you are meant to run this test to see your decorator in action.

```sh
person = Person.new(22, 'maximilianus')
person.correct_name
capitalized_person = CapitalizeDecorator.new(person)
puts capitalized_person.correct_name
capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
puts capitalized_trimmed_person.correct_name
```
-  [ ] When I ran these, I got an error in the console, I'll suggest you go through your classes and make sure everything is implemented correctly, and then run the code to make sure the result you get is what's expected. :smile: :small_airplane: :wave: 

- - Screenshot for reference:
 
![Screenshot from 2023-08-29 11-22-49](https://github.com/zuhebahmed88091/OOP_School_Library/assets/107632780/c052a22f-b84d-45e3-a6a4-a19e25908f5a)